main.rb

```diff
@@ -0,0 +1,40 @@
+require './classes/app'
```

Comment:
Nice job so far👍🏼. currently, your app is breaking because you have provided the wrong path😉. Kindly do well to add the right path and ensure your app works before submitting for a review👍🏼.
```suggestion
require './app'
```

<img width="887" alt="image" src="https://github.com/tchasinga/ruby-oop-programming/assets/64643995/435567e4-74eb-45da-83f4-254ab9b84edf">